### PR TITLE
OHCS-97 visit function wrong text length

### PR DESCRIPTION
**Pull request by SilverD3**
- Put `disease` and `diseaseType` each on a new line
- Fix 'undefined initial value' issue on fields `disease` and `diseaseType`

### CODE REVIEW

**Comment by SilverD3** [comment]
@almeida1492 Please review my PRs

File: src/components/accessories/opds/filter/styles.scss [hunk 1]
```diff
@@ -57,6 +57,9 @@
             @include susy-media($watch) {
               width: 100%;
             }
+            &.col-4 {
+              width: 100%;
+            }
```
Comment (axelpezzo) [inline src/components/accessories/opds/filter/styles.scss:62]: This is not the proper way to use col-4 as semantic class.

I suggest to change the selector name in something more "exclusive" and then fix the width that shouldn't be 33%?

**Review by axelpezzo — APPROVED** [review]

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Comment by mwithi** [comment]
@axelpezzo I don't know why, having merged this PR broke the workflow, can you check please?
https://github.com/informatici/openhospital-ui/actions/runs/4949970784

**Pull request merged**